[2022-11-30T11:18:08.415+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T11:18:05.427964+00:00 [queued]>
[2022-11-30T11:18:08.421+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T11:18:05.427964+00:00 [queued]>
[2022-11-30T11:18:08.421+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T11:18:08.421+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2022-11-30T11:18:08.421+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2022-11-30T11:18:08.430+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonSensor): waiting_for_data> on 2022-11-30 11:18:05.427964+00:00
[2022-11-30T11:18:08.433+0000] {standard_task_runner.py:55} INFO - Started process 79 to run task
[2022-11-30T11:18:08.435+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'pratice_pg', 'waiting_for_data', 'manual__2022-11-30T11:18:05.427964+00:00', '--job-id', '747', '--raw', '--subdir', 'DAGS_FOLDER/practice_pg.py', '--cfg-path', '/tmp/tmp0s56vbwy']
[2022-11-30T11:18:08.436+0000] {standard_task_runner.py:83} INFO - Job 747: Subtask waiting_for_data
[2022-11-30T11:18:08.477+0000] {task_command.py:376} INFO - Running <TaskInstance: pratice_pg.waiting_for_data manual__2022-11-30T11:18:05.427964+00:00 [running]> on host 39d0f50fcfa4
[2022-11-30T11:18:08.518+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pratice_pg
AIRFLOW_CTX_TASK_ID=waiting_for_data
AIRFLOW_CTX_EXECUTION_DATE=2022-11-30T11:18:05.427964+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-30T11:18:05.427964+00:00
[2022-11-30T11:18:08.524+0000] {python.py:68} INFO - Poking callable: <function pg_check_data at 0x7f234f639f80>
[2022-11-30T11:18:08.529+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T11:18:08.626+0000] {base.py:71} INFO - Using connection ID 'pgdb' for task execution.
[2022-11-30T11:18:08.628+0000] {base.py:213} INFO - Success criteria met. Exiting.
[2022-11-30T11:18:08.633+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=pratice_pg, task_id=waiting_for_data, execution_date=20221130T111805, start_date=20221130T111808, end_date=20221130T111808
[2022-11-30T11:18:08.651+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-11-30T11:18:08.676+0000] {taskinstance.py:2623} INFO - 1 downstream tasks scheduled from follow-on schedule check
